tensorflow/python/keras/integration_test/BUILD

```diff
@@ -2,6 +2,7 @@
 #   Contains Keras integration tests that verify with other TF high level APIs.
 
 load("//tensorflow:tensorflow.bzl", "tf_py_test")
```

Comment:
nit: load("//tensorflow:tensorflow.bzl", "tf_py_test", "cuda_py_test")

Reply:
Done!

Reply:
Looks like sanity test was not happy with this. Fixed it. Hopefully things go through this time.